src/app/_translate/lang-en.ts

```diff
@@ -4,226 +4,201 @@
 */
 export const LANG_EN_NAME = 'en';
 
-export const LANG_EN_TRANS = {
-  accountrole: 'Account role',
-  addworkflow: 'Add workflow info',
-  'add-dataset-id': 'Add dataset id',
+export const LANG_EN_STATUS_TYPES = {
+  CANCELLED: 'Cancelled',
+  FAILED: 'Failed',
+  FINISHED: 'Finished',
+  INQUEUE: 'In queue',
+  RUNNING: 'Running'
+};
+
+export const LANG_EN_TRANS = Object.assign(LANG_EN_STATUS_TYPES, {
```

Comment:
This copies all the constants in LANG_EN_STATUS_TYPES, which is not what we want and not what users of this structure would be expecting.

Reply:
And your last commit solved it.